spec-0000/index.md

```diff
@@ -32,8 +32,8 @@ All versions refer to feature releases (i.e., Python 3.8.0, NumPy 1.19.0; not Py
 
 Specifically, we recommend that:
 
-1. Support for Python versions be dropped **3 years** after their initial release.
-2. Support for core package dependencies be dropped **2 years** after their initial release.
+1. Support for Python versions be dropped **3 years** after the next version of Python is released.
+2. Support for core dependency versions be dropped **2 years** after the next version of the dependency is released.
```

Comment:
I think that is weird. Ipython releases every month, so that changes almost nothing. For a project that releases every year that extends support for a year. I don't think a project support should depends on the future release cadence that may or may not change. 

This also directly go against the principle of having a writing on the wall when dealing with companies to have a predictable end of life. I can see people getting presured "please don't release because we still need n-1 to be supported longer"